src/main/java/uk/gov/companieshouse/company_appointments/serialization/LocalDateTimeSerializer.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.companieshouse.company_appointments.serialization;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class LocalDateTimeSerializer extends JsonSerializer<LocalDateTime> {
+
+    @Override
+    public void serialize(LocalDateTime localDateTime, JsonGenerator jsonGenerator,
+                          SerializerProvider serializerProvider) throws IOException {
+        if (localDateTime == null) {
+            jsonGenerator.writeNull();
+        } else {
+            DateTimeFormatter dateTimeFormatter = DateTimeFormatter
```

Comment:
`DateTimeFormatter` is thread-safe so it can be declared as a static variable and initialised inline.

Reply:
Sonar complains but done.

Reply:
The whole DateTimeFormatter can be initialised in one shot
```
private static final DateTimeFormatter dateTimeFormatter..ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```